reflectionclient/reflectionclient.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package reflectionclient wraps other libraries to make client-less grpc calls easily.
+package reflectionclient
```

Comment:
Can we rename the package to `grpccall`? Go itself also has reflection so it's a bit hard to tell what reflection it's about.

Reply:
Yeah, also would recommend improving the README. It doesn't say what a reflectionclient does, just how to call it

Reply:
Better yet add the info in README to godocs